src/collections/Menu/Menu.js

```diff
@@ -153,7 +153,7 @@ class Menu extends Component {
     const { activeIndex } = this.state
 
     return _.map(items, (item, index) => {
-      return createShorthand(MenuItem, val => ({ content: val }), item, {
+      return createShorthand(MenuItem, val => ({ content: val, name: val }), item, {
```

Comment:
When using shorthand items as an array of strings, the strings were not passed to the `name` prop of the menu item, so the callback name was `undefined`.  I've added the literal shorthand values to the name prop so the callbacks contain the name.